packages/dds/tree/src/tree/treeTextFormat.ts

```diff
@@ -93,16 +98,23 @@ export function getGenericTreeField<T>(node: GenericTreeNode<T>, key: FieldKey,
     return newField;
 }
 
+export const enum FieldScope {
+    local = "fields",
+    // TODO: separate this from local fields by giving it a different name.
```

Comment:
Is this comment up to date? I can't tell what it means by "different name".

Reply:
Yes. Currently local and global fields both are stored in a property named "fields" (and thus not actually separate) since these two enum values have the same value.

Reply:
I added some clarification.

Reply:
Oh, that makes sense. Thanks!